runtime/tutor.txt

```diff
@@ -416,6 +415,296 @@ _________________________________________________________________
+
+ Macros are a way to record a set of actions you want to repeat.
+
+ Type Q to start recording a macro, you should see a popup at
+ the bottom of your screen. Type Q again to stop recording.
+ Type q to repeat your recorded macro.
+
+ 1. Move the cursor to the first line below marked -->.
+    Ensure your cursor is on the > of the arrow.
+ 2. Type Q to start recording.
+ 3. Edit the line to look like the bottom one.
+ 4. Exit insert and Type Q again to stop recording.
+ 5. Move to the line below and put your cursor on the > again.
+ 6. Type q to repeat the macro.
+
+ --> ... sentence doesn't have it's first and last ... .
+ --> ... sentence doesn't have it's first and last ... .
+     This sentence doesn't have it's first and last word.
+
+=================================================================
+=                        CHAPTER 4 RECAP                        =
+=================================================================
+
+ * Type y to yank (copy) text and p to paste.
+   * Type space-Y and space-P to yank/paste on the system
+     clipboard.
+
+ * Type ~ to alternate case of selected letters.
+   * Use ` and alt-` to set the case of selected layers to
+     upper and lower respectively.
+
+ * Type Q to record and stop recording a macro.
+   * Type q to repeat the recorded macro.
+
+
+
+
+
+
+
+
+=================================================================
+=                    5.1 USING THE JUMPLIST                     =
+=================================================================
+
+ Helix can keep track of "jumps" which are big movements, like
+ jumping to the definition of a function in code. It stores
+ these in what's called the jumplist.
+
+ Type C-s (ctrl-s) to manually save your current position to
+ the jumplist.
+
+ Type C-i and C-o to move forward and backwards in the jumplist
+ respectively.
+
+ 1. Type C-s somewhere.
+ 2. Move far away in the file.
+ 3. Type C-o (just once!) to come back to where you saved.
+
+
+
+
+
+=================================================================
+=                     5.2 SEARCHING IN FILE                     =
+=================================================================
+
+ Type / to search forward in file, enter to confirm search.
+ Type n to go to the next search match.
+ Type N to go to the previous search match.
+
+ 1. Type / and type in a common word, like 'banana'.
+ 2. Type enter to confirm the search.
+ 3. Use n and N to cycle through the matches.
+
+ Like the select command, searching also uses regex.
+
+ Note: To search backwards, type ? (shift-/).
+
+ Note: Unlike Vim, N doesn't change the search direction.
+       N always goes backwards and n always goes forwards.
+
+
+
+
+=================================================================
+=                        CHAPTER 5 RECAP                        =
+=================================================================
+
+ * Type C-s to save position to the jumplist.
```

Comment:
Was looking at this and I recall https://github.com/helix-editor/helix/pull/2115 which added the same function for normal mode but under different key, I wonder if `ctrl-s` is a better choice than `alt-u` now.

Reply:
Is this a request for a change or just a thought? #2115 Is about undoing, not the jumplist, AFAICT.